/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/accelerator_connector.py:317: LightningDeprecationWarning: Passing ddp_sharded `strategy` to the `plugins` flag in Trainer has been deprecated in v1.5 and will be removed in v1.7. Use `Trainer(strategy=ddp_sharded)` instead.
  rank_zero_deprecation(
/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/loops/utilities.py:91: PossibleUserWarning: `max_epochs` was not set. Setting it to 1000 epochs. To train without an epoch limit, set `max_epochs=-1`.
  rank_zero_warn(
GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Traceback (most recent call last):
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 721, in _call_and_handle_interrupt
    return self.strategy.launcher.launch(trainer_fn, *args, trainer=self, **kwargs)
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/strategies/launchers/subprocess_script.py", line 93, in launch
    return function(*args, **kwargs)
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 802, in _fit_impl
    self._data_connector.attach_data(
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/data_connector.py", line 160, in attach_data
    self._copy_trainer_model_properties(model)
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/data_connector.py", line 163, in _copy_trainer_model_properties
    ref_model = self.trainer.lightning_module or model
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 2151, in lightning_module
    return self.strategy.lightning_module
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/strategies/sharded.py", line 131, in lightning_module
    raise MisconfigurationException(
pytorch_lightning.utilities.exceptions.MisconfigurationException: `DDPShardedStrategy` requires `fairscale` to be installed. Install it by running `pip install fairscale`.
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "pl.py", line 63, in <module>
    trainer.fit(model, train_loader, val_loader)
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 770, in fit
    self._call_and_handle_interrupt(
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 737, in _call_and_handle_interrupt
    self._call_callback_hooks("on_exception", exception)
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1619, in _call_callback_hooks
    pl_module = self.lightning_module
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 2151, in lightning_module
    return self.strategy.lightning_module
  File "/home/ingu627/gtx2080/lib/python3.8/site-packages/pytorch_lightning/strategies/sharded.py", line 131, in lightning_module
    raise MisconfigurationException(
